src/main/java/jenkins/plugins/openstack/compute/internal/Openstack.java

```diff
@@ -128,8 +128,7 @@ private Openstack(@Nonnull String endPointUrl, boolean ignoreSsl, @Nonnull Opens
                 .useRegion(region);
 
         clientProvider = ClientProvider.get(client, region, config);
-        debug("Openstack client created for \"{1}\", \"{2}\".", auth.toString(), region);
-
+        debug("Openstack client created for \"{0}\", \"{1}\".", auth.toString(), region);
```

Comment:
This was the bug that lead me to create this PR - I was seeing malformed log messages saying, e.g. `Openstack client created for "RegionOne", "{2}".`
The first argument should be referenced by `{0}` and the second by `{1}` - indexes start at zero, not at one.